rules/windows/process_creation/proc_creation_win_susp_copy_browser_data.yml

```
@@ -31,9 +32,11 @@ detection:
               - ' mi '
               - ' mv '
         - Image|endswith:
+              - '\esentutl.exe' # cmd.exe /Q /c esentutl.exe /y "C:\Users\<username>\AppData\Roaming\Mozilla\Firefox\Profiles\<firefox_profile_id>.default- release\key4.db" /d C:\Users\<username>\AppData\Roaming\Mozilla\Firefox\Profiles\<firefox_profile_id>.default-release\key4.db.tmp”
```

```suggestion
              - '\esentutl.exe' # akira ransomware
```
to keep comment simple as  the link is in the references sectio,